scripts/run_locust_capacity.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+poetry run locust -f tests/locust/tests/capacity_test.py --html tests/locust/results/capacity_test.html --headless --host $HOST_URL || true;
```

Comment:
we could replace these scripts with a single script called run_locust_test.sh which takes in an argument for the test name - then using that arg to determine the path to the test file and the report file name?